hawkular-rest-status/src/main/java/org/hawkular/commons/rest/status/ManifestUtil.java

```diff
@@ -54,6 +62,6 @@
                 }
             }
         }
-        return ret;
+        return Collections.unmodifiableMap(ret);
```

Comment:
I'm -1 on this. This method returns a new map instance every time so no state is going to be modified if the caller of this method updates the map afterwards. There is therefore no reason, IMHO, to wrap the instance in an unmodifiable wrapper - it does not add any benefit to the `ManifestUtil` class nor the caller.